style.css

```diff
@@ -84,3 +85,615 @@
   left: 0;
   top: 472px;
 }
+
+.workSection {
+  display: flex;
+  flex-direction: column;
+}
+
+.workSection h1 {
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  top: 672px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+  position: absolute;
+}
+
+.imgCard {
+  position: absolute;
+  width: 320px;
+  height: 186px;
+  left: 0;
+  top: 754px;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  margin-top: 5px;
+}
```

Comment:
[OPTIONAL] Hello @solorics, I don't know why no other reviewer told you this but I've noticed that you used a fixed width for your project. Please take into account that, although this is the width displayed on the design, you shouldn't use a fixed width for your projects (it is just the width of the device used for the screenshot). I will explain: 

The idea of Responsive Web Design is that your webpage should adapt itself to any device/window size that opens it. In other words, it is responsive. This means that using a fixed width for your sections will break this responsiveness, causing a static view to be formed. So, what can you do about it? Simple, use percentages! Instead of `width: 320px`, use `width: 100%` or vw (view width) `width: 100vw`. This will make your project stretches to any window size that you can imagine 😉. If you need a smaller section, use a lower percentage. If you really need to use a fixed width, keep in mind that it may break the responsiveness of your webpage 😄! 

_But, my mobile version looks horrible stretched to a big window 😢._ 

Don't worry about this! There is a thing called [media query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) that will allow you to add different styling to different windows sizes in the future. Then, you can edit a mobile version and a desktop version for your apps, depending on the window size 😄. But for now, you should focus on the mobile version 😉. 

With a fixed width, even on mobile screens, it doesn't fit correctly. Check your project on an iPhone SE:
![image](https://user-images.githubusercontent.com/42799579/198423270-da57a92c-672b-4807-b12f-239480191d62.png)

If you are interested to learn more about it, check [this link](https://www.w3schools.com/html/html_responsive.asp) and this [youtube video](https://www.youtube.com/watch?v=VQraviuwbzU). 

Good luck and keep coding 😃!

Reply:
> worked on the width as suggested to get a responsive mobile view.
![Screenshot 2022-10-28 at 13 51 28](https://user-images.githubusercontent.com/37203742/198592252-0fcf6b64-3517-4ad4-892e-0ccdef4fbb9d.png)
![Screenshot 2022-10-28 at 13 52 12](https://user-images.githubusercontent.com/37203742/198592334-5c8979c6-f3a3-4cd3-8cef-a22170596066.png)

> My mobile view on iPhone SE.